airflow/api_connexion/openapi/v1.yaml

```diff
@@ -412,25 +412,50 @@ paths:
         - $ref: '#/components/parameters/PageOffset'
         - $ref: '#/components/parameters/OrderBy'
         - $ref: '#/components/parameters/FilterTags'
-        - name: only_active
+        - $ref: '#/components/parameters/OnlyActive'
+        - name: dag_id_pattern
           in: query
           schema:
-            type: boolean
-            default: true
+            type: string
           required: false
           description: |
-            Only return active DAGs.
+            If set, only return DAGs with dag_ids matching this pattern.
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DAGCollection'
+        '401':
+          $ref: '#/components/responses/Unauthenticated'
 
-            *New in version 2.1.1*
```

Comment:
This shouldn't be removed or am I missing something? Looks like it should be at 3814 or in a description field under `/dags`